tutorial-week-5.md

```diff
@@ -0,0 +1,88 @@
+##Website Layout Using HTML5
```

Comment:
You might call this _Semantic_ HTML. `<div>`s are part of HTML5 - just not _semantic_.